Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "cargo-edit"
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
We could probably diff the lock file before and after `cargo test`. (I think someone had plans for a cargo-tidy project but I'm not sure if anything actually got implemented)

FYI, @ordian, I sent you an invite to become a collaborator on this project. This'll give you official review and merge capacities. You've earned it :)